ts-client/src/amm/index.ts

```diff
@@ -624,6 +624,30 @@ export default class AmmImpl implements AmmImplementation {
     return this.poolState.fees.tradeFeeNumerator.mul(new BN(10000)).div(this.poolState.fees.tradeFeeDenominator);
   }
 
+  get depegTokens(): TokenInfo[] {
```

Comment:
There will be only one token is depegged, so we no need to return array

Reply:
but there will also be a chance 2 token got depeg?

Reply:
Only 1 token can be depgged based on the formula